lib/transformers/transformer.dart

```diff
@@ -12,7 +12,7 @@ abstract class Transformer {
   String get name;
 
   // The name of the class that will be generated
-  String get className => '${name[0].toUpperCase()}${name.substring(1)}Tokens';
+  String get className => '${name.pascalCase}Tokens';
```

Comment:
This change belongs in a different PR.

Reply:
I will split them then

Reply:
Sorry I was reviewing it like I was at work

Reply:
No worries, I was cutting corners because I want this version to be published haha. Should I keep it like that?